samza-core/src/main/java/org/apache/samza/zk/ScheduleAfterDebounceTime.java

```diff
@@ -48,7 +48,7 @@
   // Action name when the Processor membership changes
   public static final String ON_PROCESSOR_CHANGE = "OnProcessorChange";
 
-  public static final int DEBOUNCE_TIME_MS = 2000;
+  public static final int DEFAULT_DEBOUNCE_TIME_MS = 2000;
```

Comment:
yep, looks better. I will keep it in JobConfig, because, in theory, this value may be needed also for non ZK coordination.